manager-pullrequest.js

```diff
@@ -1,10 +1,11 @@
 const redis = require('./redis');
 const postal = require('postal');
-const prefix = 'pullrequest';
+
+const PREFIX = 'pullrequest';
```

Comment:
Rename it everywhere if you like to refactor code. This way it's not the same in the other manager classes and it drives me crazy 🤪